addons/portal/static/src/chatter/portal/portal_chatter_service.js

```diff
@@ -49,6 +49,9 @@ export class PortalChatterService {
 
     async initialize(env) {
         const chatterEl = document.querySelector(".o_portal_chatter");
+        if (!chatterEl) {
+            return;
+        }
```

Comment:
What case is this extra condition covering?

Reply:
There is no `o_portal_chatter` in project sharing, so the portal chatter service will crash if it cannot find the related element. In the previous code this service wasn't in the project sharing bundle. Another way is to remove it from the project sharing bundle.

Reply:
Yes then I would move the file out of the `portal` bundle. I can be top level and as a single line in the portal manifest (same comment as the boot file basically).